coverage.py

```diff
@@ -463,101 +485,18 @@ def output_source(self):
         """
         return DataSource.lookup(self._db, self.output_source_name)
 
-    def license_pool(self, identifier):
-        """Finds or creates the LicensePool for a given Identifier."""
-        license_pool = identifier.licensed_through
-        if not license_pool:
-            if self.CAN_CREATE_LICENSE_POOLS:
-                # The source of this data also provides license
-                # pools, so it's okay to automatically create
-                # a license pool for this book.
-                license_pool, ignore = LicensePool.for_foreign_id(
-                    self._db, self.output_source, identifier.type, 
-                    identifier.identifier
-                )
-            else:
-                return None
-        return license_pool
-
```

Comment:
This got moved to CollectionCoverageProvider. Normal CoverageProviders aren't supposed to call it.

(Maybe normal CoverageProviders shouldn't store a Collection at all?)